Simplification of the supported controllers

DubhAd opened the pull request:
Stripped it down to the key fact - *static controllers* are all that'll work with OZW.

Noted that Z-Wave Plus controllers are recommended, to take advantage of all the features of Z-Wave Plus.

Noted that S2 and Smart Start aren't supported by OZW

DubhAd committed:
Simplification of the supported controllers

Stripped it down to the key fact - *static controllers* are all that'll work with OZW.

Noted that Z-Wave Plus controllers are recommended, to take advantage of all the features of Z-Wave Plus.

Noted that S2 and Smart Start aren't supported by OZW

balloob commented on source/_docs/z-wave/controllers.markdown:
```diff
@@ -11,22 +11,9 @@ footer: true
-
-| Device                    | Works on Linux | Works on Windows | Works on OSX |
-|---------------------------|----------------|------------------|--------------|
-| Aeotec Z-Stick            |   &#10003;     |                  |              |
```
Let's keep a list of controllers that work. People want to see an easy list and we already have that info.

DubhAd committed:
Controllers restored

Restored the list of confirmed sticks, but taken out the lists of OSs because we just took the list from the OZW pages, and it's clear that's outdated and incomplete

frenck commented:
Thanks @DubhAd 👍
